data/settings/src/main/java/com/google/jetpackcamera/settings/model/CaptureModeStatus.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.jetpackcamera.settings.model
+
+enum class CaptureModeStatus {
```

Comment:
I think thats ok, though we should be mindful of an auto generated class in the build folder (from proto-datastore) that has the same name, `CaptureMode`

Reply:
I think it's better for all of them to have the same name, since they're all essentially referring to the same thing.

When we need to specify them by type in the same file, we can use an import as alias if needed.